testsuite/MDAnalysisTests/analysis/test_persistencelength.py

```diff
@@ -92,6 +92,11 @@ def test_plot_with_ax(self, p_run):
         ax2 = p_run.plot(ax=ax)
 
         assert ax2 is ax
+    
+    def test_current_axes(self, p_run):
+        ax = plt.figure()
```

Comment:
`plt.figure()` returns a matplotlib Figure, not Axes. This test will always pass, but for the wrong reasons. If you look just above you can see that the previous test used ``fig, ax = plt.subplots()`` -- that might be a better option.

Reply:
Updated. Attached is screenshot of test passed.
![test](https://user-images.githubusercontent.com/32260593/76267008-58548180-628f-11ea-9d50-746e9e65dc03.png)